pyqtgraph/graphicsItems/ViewBox/tests/test_ViewBox.py

```diff
@@ -1,19 +1,17 @@
 #import PySide
 import pyqtgraph as pg
+import pytest
 
-app = pg.mkQApp()
-qtest = pg.Qt.QtTest.QTest
+QRectF = None
+app = None
+win = None
+vb = None
 
-def assertMapping(vb, r1, r2):
-    assert vb.mapFromView(r1.topLeft()) == r2.topLeft()
-    assert vb.mapFromView(r1.bottomLeft()) == r2.bottomLeft()
-    assert vb.mapFromView(r1.topRight()) == r2.topRight()
-    assert vb.mapFromView(r1.bottomRight()) == r2.bottomRight()
-
-def test_ViewBox():
-    global app, win, vb
+def setup_module():
```

Comment:
With this setup/teardown, the test results are order-dependent (because the view mapping may change depending on the state of the viewbox from the prior test). So I think either each test needs to create its own window+viewbox, or they need to stay as a single function to ensure their order is fixed.